docs/attribution.md

```diff
@@ -1,18 +1,22 @@
 # Attribution
 
-Attribution is required for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) page.
+Attribution is required for using Mapzen's hosted terrain service as well as many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) and the [Mapzen rights](https://mapzen.com/rights) pages.
 
 **Required attribution:**
 
-> Mapzen terrain tiles contain content courtesy of the U.S. Geological Survey and U.S. National Oceanic and Atmospheric Administration.
+**Using Mapzen's hosted service:**
```

Comment:
I agree on using the simplest terms possible (If people are self-hosting and are concerned about proper attribution in that case, we can deal with them on a small basis).